include/Types.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
Can we name this `DeviceTypes.h` or something? `Types.h` is pretty ambiguous

Reply:
Sure